e2e/updatecli.d/success.d/command.yaml

```diff
@@ -24,7 +24,7 @@ sources:
 #    name: Should trigger an error
 #    kind: shell
 #    spec:
-#      command: "/bin/donotexit"
+#      command: "/bin/nonexistent"
```

Comment:
Before this change, there were a handful of different spellings used: https://github.com/updatecli/updatecli/pull/1330/commits/5a658bb9a94bbe0e36715c429c832bfda09016f3

Standardizing on a single spelling makes it easier to search for related items.

It's possible this specific case really meant `do-not-exit`, but that's pretty unlikely, so this specific case is almost certainly an error (it's harmless as the effect would be the same as `donotexit` is also unlikely to exist, and of course, this is a comment).

Reply:
Thanks indeed this one is a commented example of a script which shouldn't work